NAMESPACE

```diff
@@ -4,13 +4,15 @@ export(LahtiMData)
 export(LahtiMLData)
 export(LahtiWAData)
 export(OKeefeDSData)
+export(SailaniIPOPData)
 export(SilvermanAGutData)
 export(SongQAData)
 export(artificialgut)
 export(atlas1006)
 export(dietswap)
 export(peerj32)
 export(qa10934)
+export(sailaniipop)
```

Comment:
we can omit this alias can we? it is almost the same as the main function name.

Reply:
Yes